jekyll/_ccie/Artifactory.md

```diff
@@ -0,0 +1,61 @@
+# Artifactory 
+
+## Deploy
+
+Artifactory has great documentation explaining how to leverage their [REST API](https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API).
+
+We figure we use this space to highlight some sample projects that will show you how to best leverage using CircleCI and Artifactory together. 
+
+It is super important for you to have created your Repository before starting this example, otherwise CircleCI will have nowhere to store your dependencies.
```

Comment:
Suggested:

"Ensure that you've created your repository before starting this example, otherwise CircleCI won't have a place to store your dependencies."